notebook/static-src/notebook/js/notebook.js

```diff
@@ -17,7 +16,7 @@ define(function (require) {
     var configmod = require('services/config');
     var session = require('services/sessions/session');
     var celltoolbar = require('notebook/js/celltoolbar');
-    var marked = require('components/marked/lib/marked');
+    var marked = require('marked');
```

Comment:
this is one of the components that travis is whining about, hopefully it's not this line causing it